src/scripts/generate-unary-ops.py

```diff
@@ -0,0 +1,218 @@
+#!/usr/bin/env python
+import sys
+import os
+import settings
+import collections
+import random
+import subprocess
+path = os.path.join(settings.TEMPLATE_DIR)
+sys.path.append(path)
+import validate
+import messages
+from parseargs import parse_args
+
+GENERATED_FILENAME = '''unary-ops-{NUMBER}.js''' # The file name prefix of the generated tests
```

Comment:
Hide these variables with '_' prefix. Maybe you can declare them as a local variable if you are not about to export them.